.clang-format

```diff
@@ -37,7 +37,7 @@ BraceWrapping:
   SplitEmptyRecord: true
   SplitEmptyNamespace: true
 BreakBeforeBinaryOperators: None
-BreakBeforeBraces: Attach
+BreakBeforeBraces: Custom # Modified
```

Comment:
This seems really inconsistent now. Is there a reason you made it custom?

Reply:
Yes. To allow breaking before braces after functions. These breaks are widely used in the current code bode.